### ISSUE TITLE
Assigning ID, classes, or data attributes via schema

### ISSUE THREAD

**Issue by ciocarlia1981** [jdorn/json-editor#643]
Is there any way to assign an ID, class, or attribute to an editor via the schema or the options? 
For example, something like:

{
  "title": "Name",
  "type": "string",
  "options": {
    "id":"field_name",
    "class":"highlighted_field"
  }
}

If not, what would be the most expedient way to adjust the code to enable such a feature?

(PS - thank you for the work on this very helpful library!)

**Comment by sinanyalcinkaya** [comment]
I am looking for the way too...

**Comment by TimurKady** [comment]
I agree! Very strong need to add a "class" option to a non-object node. May be directly, e.g.:
`{
    title: "It's my title",
    type: "string",
    class: "my_class_name"
}
`
